lib/app/features/competiton/searched_results/result_details/widgets/tabs/stats_tab.dart

```diff
@@ -1,49 +1,47 @@
 import 'package:flutter/material.dart';
 import 'package:score_live/app/common/extensions/context/applocalization_context.dart';
 import 'package:score_live/app/common/extensions/enums.dart';
-import 'package:score_live/presentation/constants/app_colors.dart';
 import 'package:score_live/presentation/constants/app_const_variables.dart';
 import 'package:score_live/presentation/constants/text_styles.dart';
 
-import '../../../../../../../models/topscorers_model/topscorers_model.dart';
+import '/models/topscorers_model/top_scorers_model.dart';
+import 'stats_sub_tabs/top_goals_sub_tab.dart';
 
 class StatsTab extends StatelessWidget {
-  const StatsTab({super.key, required this.topScorers, required this.statsSwitch});
+  const StatsTab({super.key, required this.topGoals, required this.topAssists, required this.statsSwitch});
 
   final StatsSwitch statsSwitch;
-  final List<TopScorersModel> topScorers;
+  final List<TopScorersModel> topGoals;
+  final List<TopScorersModel> topAssists;
 
   @override
   Widget build(BuildContext context) {
-    if (topScorers.isEmpty) {
-      return SizedBox(
-        height: 200,
-        child: Center(
-          child: Text(
-            context.localizations.noStats,
-            textAlign: TextAlign.center,
-            style: const CustomTextStyle(color: Colors.white, fontSize: 24, fontWeight: FontWeight.w700),
-          ),
-        ),
-      );
-    }
-    return Column(
-      children: [
-        const StatsHeaderWithSwitch(),
-        Builder(
-          builder: (context) => switch (statsSwitch) {
-            StatsSwitch.goals => Container(
-                padding: const EdgeInsets.all(12.0),
-                color: AppColors.listTileGrey,
-                child: const Column(
-                  //TODO Display data provided by Api here
-                  children: [],
-                ),
-              ),
-            StatsSwitch.assists => const SizedBox(),
-          },
-        )
-      ],
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 12.0),
+      child: Column(
+        children: [
+          const StatsHeaderWithSwitch(),
+          const SizedBox(height: 20),
+          topGoals.isEmpty
+              ? SizedBox(
+                  height: 200,
+                  child: Center(
+                    child: Text(
+                      context.localizations.noStats,
+                      textAlign: TextAlign.center,
+                      style: const CustomTextStyle(color: Colors.white, fontSize: 24, fontWeight: FontWeight.w700),
+                    ),
+                  ),
+                )
+              : Builder(
```

Comment:
Why do you need to use builder here?

Reply:
you're right, it unnecessary, I'm removing it